Stripe/UI/STPPaymentCardTextField.m

```diff
@@ -403,6 +403,55 @@ - (STPCardParams *)card {
     return c;
 }
 
+- (void)setCard:(nonnull STPCardParams *)card {
+    if (card.number.length) {
```

Comment:
Won't this break if you pass e.g. a card with an empty number to a field where the number is already set? (same with expiration and cvc below.)